### PR TITLE
feat: Enhance add fragment dialog with warning message

### PR DESCRIPTION
**Pull request by nikmace**
Feat for #1684.
- Enhances the AddFragment.fragment.xml with a warning message.
- Checks for active commands when opening the dialog.

### CODE REVIEW

File: packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts [hunk 1]
```diff
@@ -296,5 +321,7 @@ export default class AddFragment extends BaseDialog {
         );
 
         await this.commandExecutor.pushAndExecuteCommand(command);
+
+        await this.rta._serializeToLrep();
```
Comment (tobiasqueck) [inline packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts:325]: Wouldn't that send all unsaved changes to LREP, not just the fragment change?

Comment (nikmace) [reply packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts:325]: Yes, but that is the idea, before that we are showing a warning message to the user that all his unsaved changes will be saved.

File: packages/preview-middleware-client/src/adp/ui/AddFragment.fragment.xml [hunk 2]
```diff
@@ -50,6 +50,18 @@
                 </Input>
             </f:content>
         </f:SimpleForm>
+        <VBox 
+            class="sapUiSmallMarginBeginEnd" 
+            visible="false">
+            <items>
+                <MessageStrip
+                    text="Have in mind that adding this change will also save the changes you have made to your project that are yet unsaved."
+                    type="Warning"
```
Comment (GDamyanov) [inline packages/preview-middleware-client/src/adp/ui/AddFragment.fragment.xml:59]: Is this text synced with Simona?

File: packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts [hunk 3]
```diff
@@ -57,11 +58,35 @@ export default class AddFragment extends BaseDialog {
...
+        }
+    }
+
+    /**
+     * Displays a warning message to the user by making the warning VBox visible
+     * within the dialog's content.
+     */
+    private showWarningMessage(): void {
+        const warningBox = this.dialog.getContent()[1] as VBox;
+        warningBox.setVisible(true);
```
Comment (GDamyanov) [inline packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts:87]: Is there a possibility "this.dialog.getContent()[1]" to be undefined?

**Review by GDamyanov — CHANGES_REQUESTED** [review]

**Comment by nikmace** [comment]
Closed in favor of #1691.

**Pull request closed without merging**